draft-ietf-rtcweb-jsep.xml

```diff
@@ -1977,13 +1977,205 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+            stop and an error MUST be returned.</t>
+            <t>Next, the SessionDescription is parsed into a data structure,
+              as described in the
+              <xref target="sec.parsing-a-desc"/> section below.
+            </t>
+            <t>Finally, the parsed SessionDescription is applied as described in
+              the <xref target="sec.applying-a-local-desc"/> section below.</t>
+          </list>
+        </t>
+      </section>
 
-      <section title="Parsing an Answer" anchor="sec.parsing-an-answer"></section>
+      <section title="Processing a Remote Description" anchor="sec.processing-a-remote-desc">
+        <t>When a SessionDescription is supplied to setRemoteDescription, the
+            following steps MUST be performed:
+          <list style="symbols">
+            <t>First, the type of the SessionDescription is checked against the
+                  current state of the PeerConnection:
+              <list style="symbols">
+                <t>If the type is "offer", the PeerConnection state MUST be
+                      either "stable" or "have-remote-offer".</t>
+                <t>If the type is "pranswer" or "answer", the PeerConnection
+                      state MUST be either "have-local-offer" or
+                      "have-remote-pranswer".</t>
+              </list>
+            </t>
+            <t>If the type is not correct for the current state, processing MUST
+            stop and an error MUST be returned.</t>
+            <t>Next, the SessionDescription is parsed into a data structure,
+              as described in the
+              <xref target="sec.parsing-a-desc"/> section below.
+            </t>
+            <t>Finally, the parsed SessionDescription is applied as described in
+              the <xref target="sec.applying-a-remote-desc"/> section below.</t>
+          </list>
+        </t>
+      </section>
 
-      <section title="Applying a Local Description" anchor="sec.applying-a-local-descript"></section>
+      <section title="Parsing a Session Description" anchor="sec.parsing-a-desc">
+        <t>When a SessionDescription of any type is supplied to
+        setLocal/RemoteDescription, the implementation must parse it and
+        reject it if it is invalid. The exact details of this
+        process are explained below.</t>
+        <t>The SDP contained in the session description object consists of a
+        sequence of text lines, each containing a key=value expression.
+        The SDP is read,
+        line-by-line, and converted to a data structure that contains the
+        deserialized information. However, SDP supports various fields that are not relevant to JSEP applications, and so these fields can be ignored.
+        For each line, the implementation will first ensure it is syntactically
+        correct, and then either parse the data to store for later use,
+        or discard the line and move on. If the line is not well-formed, or
```

Comment:
To be clear, I think you're saying that lines which are to be discarded must still be syntactically valid, right?